src/main/resources/META-INF/rewrite/raise-change-type.yml

```diff
@@ -15,3 +15,4 @@ recipeList:
     - org.openrewrite.java.ChangeType:
         oldFullyQualifiedTypeName: arrow.core.continuations.EagerEffectScope
         newFullyQualifiedTypeName: arrow.core.raise.Raise
+    - arrow.RaiseAddImport
```

Comment:
The imports aren't added because `ChangeType` updates the types from `Effect(X)` to `Raise`, so the types don't exist when `RaiseAddImport` is executed.

To address this, either move the `arrow.RaiseAddImport` recipe before the `ChangeType` recipes, or check for the `changed` types instead. In both cases, use set onlyIfReferenced to `true`.